crates/chia-datalayer/src/merkle.rs

```diff
@@ -398,8 +398,12 @@ fn get_free_indexes_and_keys_values_indexes(
 #[derive(Debug)]
 pub struct MerkleBlob {
     blob: Vec<u8>,
+    // TODO: would be nice for this to be deterministic ala a fifo set
     free_indexes: HashSet<TreeIndex>,
     key_to_index: HashMap<KvId, TreeIndex>,
+    // TODO: used by fuzzing, some cleaner way?  making it cfg-dependent is annoying with
+    //       the type stubs
+    pub check_integrity_on_drop: bool,
```

Comment:
would you have the same type-stub issue if you made it depend on `debug_assertions` configuration? i.e. debug builds.

Reply:
i think that a `maturin develop` was perhaps leading to the compilcation with this present while the wheel build did not.  maybe.  i had it on `#[cfg(any(test, debug_assertions))]` which is the enabler for the drop implementation itself.  the fuzzer docs also talk about a fuzzer cfg though using it triggered errors.  i do want to revisit this to better understand the various cases and make it cleaner.